### PR TITLE
Add feature summary to CMake

### PR DESCRIPTION
**Pull request by FlexW**
Signed-off-by: Felix Weilbach <felix.weilbach@nextcloud.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,9 @@ set(CMAKE_CXX_STANDARD 14)
 
 project(client)
 
+include(FeatureSummary)
+feature_summary(WHAT REQUIRED_PACKAGES_NOT_FOUND FATAL_ON_MISSING_REQUIRED_PACKAGES)
```
Comment (mgallien) [inline CMakeLists.txt:7]: Remove this instance of feature_summary, you need to call it only once at the end.

File: CMakeLists.txt [hunk 2]
```diff
@@ -274,3 +277,5 @@ elseif(BUILD_CLIENT)
     install( FILES sync-exclude.lst DESTINATION ${SYSCONFDIR}/${APPLICATION_SHORTNAME} )
     configure_file(sync-exclude.lst bin/sync-exclude.lst COPYONLY)
 endif()
+
+feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
```
Comment (mgallien) [inline CMakeLists.txt:281]: please add INCLUDE_QUIET_PACKAGES

**Review by mgallien — COMMENTED** [review]
See my comments

**Comment by FlexW** [comment]
> See my comments

Done

**Review by mgallien — APPROVED** [review]
Thanks

**Pull request merged**